### PR TITLE
tidy up publish-assets

### PR DESCRIPTION
**Pull request by jetersen**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,10 +4,6 @@ BUILD_DATE=`date -R`
 VERSION=$(shell git describe --tags)
 GOVERSION=$(shell go version)
 
-ENV=updateCli.env
```
Comment (jetersen) [inline Makefile:7]: As mentioned in https://github.com/olblak/updatecli/pull/117#issuecomment-724665899 this part is not needed.

File: Makefile [hunk 2]
```diff
@@ -4,10 +4,6 @@ BUILD_DATE=`date -R`
...
-
-include $(ENV)
```
Comment (jetersen) [inline Makefile:9]: As mentioned in https://github.com/olblak/updatecli/pull/117#issuecomment-724665899 this part is not needed.

**Review by olblak — APPROVED** [review]

**Review by olblak — APPROVED** [review]
For some reason the test is not passing because it doesn't' have access to a dockerhub secret

**Comment by jetersen** [comment]
> For some reason the test is not passing because it doesn't' have access to a dockerhub secret

You need to pass the env through I think :)

**Pull request merged**